etl/steps/data/garden/countries/2023-09-25/isd.countries.json

```diff
@@ -0,0 +1,205 @@
+{
```

Comment:
There are 230 missing values in mapping. Is that a problem?

Reply:
It's fine! thanks